.github/workflows/ci.yml

```
@@ -51,7 +51,9 @@ jobs:
         with:
           toolchain: "1.86"
           components: clippy
-      - run: cargo clippy --all-targets --all-features -- -D warnings
+      - uses:  clechasseur/rs-clippy-check@v4.0.5
```

nit (here and below)
```suggestion
      - uses: clechasseur/rs-clippy-check@v4.0.5
```